src/ja/common/job.py

```diff
@@ -63,12 +63,14 @@ class Job:
     """!
     Represents a job with all of its attributes.
     @param owner_id The unix user id of the user who owns this job.
```

Comment:
unix -> Unix

Reply:
This is not a part of this PR at all. Send another pull request to fix it :)